Kenya/KEN_transform_load_dlt.py

```diff
@@ -219,7 +231,7 @@ def boost_gold():
         .withColumn('country_name', lit(COUNTRY))
         .select('country_name',
                 'year',
-                col('Initial_Budget_Printed_Estimate').alias('approved').cast(DoubleType()),
+                col('Final_Budget_Approved_Estimate').alias('approved').cast(DoubleType()),
```

Comment:
If approved and revised are set to be the same, perhaps consider aliasing revised directly from approved to save casting the data type twice.